setup.py

```diff
@@ -0,0 +1,24 @@
+from setuptools import setup, find_packages
+
+PACKAGENAME = "testingframework"
+DESCRIPTION = "Module for testing various interatomic potentials"
+AUTHOR = "Libatoms"
+AUTHOR_EMAIL = ""
+
+version = {}
+with open("version.py") as fp:
+    exec(fp.read(), version)
+
+setup(
+    name=PACKAGENAME,
+    packages=find_packages(),
```

Comment:
there is no `__init__.py` in the `testingframework` directory, that may be useful there